celery_serverless/handler.py

```diff
@@ -82,10 +84,39 @@ def watchdog(event, context):
     else:
         watched = KombuQueueLengther(queue_url, 'celery')   # TODO: Allow queue name to be chosen
 
-    Watchdog(communicator=build_intercom(intercom_url), name=lock_name, lock=lock, watched=watched).monitor()
+    watchdog = Watchdog(communicator=build_intercom(intercom_url), name=lock_name, lock=lock, watched=watched)
+
+    try:
+        remaining_seconds = context.get_remaining_time_in_millis() / 1000.0
+    except Exception as e:
+        logger.exception('Could not got remaining_seconds. Is the context right?')
```

Comment:
Typo `Could no get`

Reply:
"Não pude pegar".

Are you sure? Just tell and I will update.